kafka/assets/monitors/broker_produce_latency.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[Kafka] High produce latency on broker",
+	"type": "metric alert",
+	"query": "avg(last_4h):avg:kafka.request.produce.time.99percentile{*} by {name} > 10000",
+	"message": "The p99 produce latency on broker {{name.name}} is high.\n\nThe p99 produce latency for this broker has been high on average for\nthe last 4 hours.\n\n**Potential Impacts**\n\n  - Client timeouts\n  - Delays in the ability of clients to process their workload\n  - Could be a leading indicator that the broker is falling behind\n    due to lack of capacity or a performance-impacting incident.\n\n**Recommended Actions**\n\n  - Investigate the state of the broker\n  - Consider topic rebalancing if the traffic on a given topic has\n    outstripped the resources available to it\n  - Consider expanding capacity by adding additional brokers\n  - Broker restart or replacement can help in some situations.\n    If TCP memory is high, and increasing in correlation with the\n    load, this could mean that the disk is struggling to keep up.\n    Restarting kafka has shown some immediate benefits when it comes\n    to reducing the load.",
+	"tags": [
+		"integration:kafka"
+	],
+	"options": {
+		"notify_audit": false,
+		"locked": false,
+		"timeout_h": 0,
+		"new_host_delay": 300,
+		"require_full_window": true,
+		"notify_no_data": true,
+		"renotify_interval": "0",
```

Comment:
`renotify_interval` needs to be int. This is currently breaking ES7 migration on staging